src/main/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -45,25 +46,58 @@ package object stage {
   /*
    Logic opperators that work with Try[Boolean]
    TODO future candidate for dans-scala-lib.
+   TODO investigate why it doesn't (maven) build with the standard && and || for the case 'Boolean && Try[Boolean]'
```

Comment:
@DANS-KNAW/easy 
@PaulBoon and I have looked into this and we don't understand what is going on here. This error only occurs while generating the scala-doc (a.k.a. after all code is compiled and all tests have run). I will look into this some more ASAP (probably over the weekend) and maybe create some issues on the generator's project issue tracker.

_to be continued_